src/mods/common/analyzer/update/bpy.props.mod.rst

```diff
@@ -0,0 +1,63 @@
+.. mod-type:: update
+
+.. module:: bpy.props
+
+.. function:: EnumProperty(items, name="", description="", translation_context="*", default=None, options={'ANIMATABLE'}, override=set(), tags=set(), update=None, get=None, set=None)
+
+   Returns a new enumerator property definition.
+
+   :arg items: sequence of enum items formatted:
+      ``[(identifier, name, description, icon, number), ...]``.
+
+      The first three elements of the tuples are mandatory.
+
+      :identifier: The identifier is used for Python access.
+      :name: Name for the interface.
+      :description: Used for documentation and tooltips.
+      :icon: An icon string identifier or integer icon value
+         (e.g. returned by :class:`bpy.types.UILayout.icon`)
+      :number: Unique value used as the identifier for this item (stored in file data).
+         Use when the identifier may need to change. If the *ENUM_FLAG* option is used,
+         the values are bit-masks and should be powers of two.
+
+      When an item only contains 4 items they define ``(identifier, name, description, number)``.
+
+      Separators may be added using None instead of a tuple.
+      For dynamic values a callback can be passed which returns a list in
+      the same format as the static list.
+      This function must take 2 arguments ``(self, context)``, **context may be None**.
+
+      .. warning::
+
+         There is a known bug with using a callback,
+         Python must keep a reference to the strings returned by the callback or Blender
+         will misbehave or even crash.
+   :type items: collections.abc.Iterable[tuple[str, str, str] | tuple[str, str, str, int] | tuple[str, str, str, str, int] | None] | collections.abc.Callable[[typing.Any, :class:`bpy.types.Context` | None], collections.abc.Iterable[tuple[str, str, str] | tuple[str, str, str, int] | tuple[str, str, str, str, int] | None]]
+   :mod-option arg items: skip-refine
```

Comment:
From now, we only need this two lines.
Before commiting changes, you will need to rebase to master branch.